.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Use case
+
+<!-- What is the final use case? Describe it with words, code or even sandboxes (4ex: [codesandbox.io](https://codesandbox.io)) -->
+
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+- [x] _Bug fix_ (non-breaking change which fixes an issue)
+- [x] _New feature_ (non-breaking change which adds functionality)
+- [x] **_Breaking change_** (fix or feature that would cause existing functionality to not work as before)
```

Comment:
Shall we make these checkboxes un-selected `- [ ] ...`?

Reply:
Because it asks to delete non-relevant items - no;
I'm thinking it'll be easier to scan the PR text if PR type will be highlighted with cross.